docs/content/docs/deployment/tasks-scheduling/balanced_tasks_scheduling.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Balanced Tasks Scheduling
+weight: 5
+type: docs
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Balanced Tasks Scheduling
+
+This page describes the background and principle of balanced tasks scheduling, 
+how to use it when running streaming jobs.
+
+## Background
+
+When the parallelism of all vertices within a Flink streaming job is inconsistent,
+the [default strategy]({{< ref "docs/deployment/config" >}}#taskmanager-load-balance-mode)
+of Flink to deploy tasks sometimes leads some `TaskManagers` have more tasks while others have fewer tasks, 
+resulting in excessive resource utilization at some `TaskManagers` 
+that contain more tasks and becoming a bottleneck for the entire job processing.
+
+{{< img src="/fig/deployments/tasks-scheduling/tasks_scheduling_skew_case.svg" alt="The Skew Case of Tasks Scheduling" class="offset" width="50%" >}}
+
+As shown in figure (a), given a Flink job comprising two vertices, `JobVertex-A (JV-A)` and `JobVertex-B (JV-B)`, 
+with parallelism degrees of `6` and `3` respectively,
+and both vertices sharing the same slot sharing group.
+Under the default tasks scheduling strategy, as illustrated in figure (b), 
+the distribution of tasks across `TaskManagers` may result in significant disparities in task load. 
+Specifically, the `TaskManager`s with the highest number of tasks may host `4` tasks, 
+while the one with the lowest load may have only `2` tasks. 
+Consequently, the `TaskManager`s bearing 4 tasks is prone to become a performance bottleneck for the entire job.
+
+Therefore, Flink provides a task-quantity-based balanced tasks scheduling capability. 
+Within the job's resource view, it aims to ensure that the number of tasks 
+scheduled to each `TaskManager` is as close as possible, thereby improving the resource usage skew among `TaskManagers`.
```

Comment:
I was expecting `is as close as possible` be as `as close as possible to ...`

Reply:
nice idea! Updated.

Please help take a look , @davidradl  thanks a lot !